client.go

```diff
@@ -498,6 +529,24 @@ func bitsToImportRequest(indexName string, frameName string, slice uint64, bits
 	}
 }
 
+func statusToNodeSlicesForIndex(status *Status, indexName string) map[uint64]*URI {
```

Comment:
`NewURIFromAddress` would not fail in this case, as long as the the valid URI format is the same between the server and the client but I can return an error here too (omitting it saved us an extra test).